scrapy_poet/commands.py

```diff
@@ -109,6 +110,10 @@ def run(self, args, opts):
         type_name = args[0]
         url = args[1]
 
+        if "" not in sys.path:
+            # when running without a Scrapy project the current dir may not be in sys.path,
+            # but the user may expect modules in the current dir to be available
```

Comment:
Would it make sense to do it only if there is no Scrapy project detected, to keep it safe?